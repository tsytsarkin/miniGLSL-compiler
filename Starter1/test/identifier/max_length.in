abcdefghijklmnopqrstuvwxyz012345
abcdefghijklmnopqrstuvwxyz0123456
