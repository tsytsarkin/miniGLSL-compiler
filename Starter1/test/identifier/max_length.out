TOKEN 260 : abcdefghijklmnopqrstuvwxyz012345

LEXICAL ERROR, LINE 2: Identifier exceeds maximum length of 32 characters
